README.md

```diff
@@ -61,21 +61,21 @@ NOTE:
 | ------ | --------------------------------------------------------- | -------------- | ------------------ | ---------------- | -------------------- | ------------------ |
-| RBD    | Dynamically provision, de-provision Block mode RWO volume | GA             | >=v1.0.0           | >=v1.0.0         | >= Mimic             | >= v1.13.0         |
-|        | Dynamically provision, de-provision Block mode RWX volume | GA             | >=v1.0.0           | >=v1.0.0         | >= Mimic             | >= v1.13.0         |
-|        | Dynamically provision, de-provision File mode RWO volume  | GA             | >=v1.0.0           | >=v1.0.0         | >= Mimic             | >= v1.13.0         |
-|        | Creating and deleting snapshot                            | Alpha          | >=v1.0.0           | >=v1.0.0         | >= Mimic             | >= v1.13.0         |
-|        | Provision volume from snapshot                            | Alpha          | >=v1.0.0           | >=v1.0.0         | >= Mimic             | >= v1.13.0         |
+| RBD    | Dynamically provision, de-provision Block mode RWO volume | GA             | >= v1.0.0          | >= v1.0.0        | >= Mimic             | >= v1.13.0         |
+|        | Dynamically provision, de-provision Block mode RWX volume | GA             | >= v1.0.0          | >= v1.0.0        | >= Mimic             | >= v1.13.0         |
+|        | Dynamically provision, de-provision File mode RWO volume  | GA             | >= v1.0.0          | >= v1.0.0        | >= Mimic             | >= v1.13.0         |
+|        | Creating and deleting snapshot                            | Alpha          | >= v1.0.0          | >= v1.0.0        | >= Mimic             | >= v1.13.0         |
+|        | Provision volume from snapshot                            | Alpha          | >= v1.0.0          | >= v1.0.0        | >= Mimic             | >= v1.13.0         |
 |        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
-|        | Resize volume                                             | -              | -                  | -                | -                    | -                  |
-|        | Metrics Support                                           | -              | -                  | -                | -                    | -                  |
-| CephFS | Dynamically provision, de-provision File mode RWO volume  | Alpha          | >=v1.1.0           | >=v1.0.0         | Nautilus (>=14.2.2)  | >=v1.13.0          |
-|        | Dynamically provision, de-provision File mode RWX volume  | Alpha          | >=v1.1.0           | >=v1.0.0         | Nautilus (>=v14.2.2) | >=v1.13.0          |
+|        | Resize volume                                             | Beta           | >= v2.0.0          | >= v1.1.0        | >= Mimic             | >= v1.15.0         |
```

Comment:
You mention the version after Nautilus, maybe do so for Mimic too?

Reply:
for cephfs version is required,  not for mimic. ceph-csi works with all minor version of Mimic but require 14.2.2 in Nautilus

Reply:
Ah, now I understand. Can you then just write "Mimic (>= 13.0.0)"? For everything else it also counts as a minimal version, and it is assumed that newer Ceph versions (whatever comes after Nautilus) is also supported.

Reply:
done updated now PTAL